_topic_map.yml

```diff
@@ -2308,6 +2310,8 @@ Topics:
       File: virt-importing-virtual-machine-images-datavolumes
     - Name: Importing virtual machine images to block storage with DataVolumes
       File: virt-importing-virtual-machine-images-datavolumes-block
+    - Name: Importing container disks with DataVolumes
+      File: virt-importing-container-disk-with-datavolumes
```

Comment:
I'm not sure why this should have it's own section rather than being rolled into the `Importing virtual machine images with DataVolumes` that already exists. 

This is really similar to the existing http import documentation, The only difference is that we're using the DataVolume "registery" source instead of "http"

Reply:
You make a really good point. That was my first thought but looking at the preparation steps and concepts required for this I thought it might be best to have a complete user story for container disks - what they are, how to prep them, specific examples, etc - and to improve searchability. However I've followed up with a few people and have learnt that searchability won't be impacted, and I think I can now see a way of handling this without having too many messy optional/deviations, like "If you've got X you need to have done Y but if you've got Z then you must A." 
I'll fix up the other feedback (which will go into #25359) so I can get that PR merged and then come back to reorganising this content. We're getting close to a docs freeze but I'll try and make this work before that happens next week.